### PR TITLE
Ser 1175 extended summary

### PR DESCRIPTION
**Pull request by idradm**
## Links

* http://wikia-inc.atlassian.net/browse/SER-1175

## Description

Extends image review summary page with uploads counts and per source reviews.

## Reviewers

@Wikia/services-team

### CODE REVIEW

File: front/main/app/templates/components/image-review-summary.hbs [hunk 1]
```diff
@@ -41,25 +43,43 @@
...
-				{{#each model.summary.userStatistics as |user|}}
+			{{#if model.summary.userStatistics}}
+				<table class="image-review-modal-history">
 					<tr>
-						<td>{{user.userName}}</td>
-						<td>{{user.distanceToAvg}}</td>
-						{{#each user.reviewsCountPerStatus as |countPerStatus|}}
-							<td>{{countPerStatus.count}}</td>
+						<td>User name</td>
+						<td>Distance to avg</td>
```
Comment (sqreek) [inline front/main/app/templates/components/image-review-summary.hbs]: can we remove it?

Comment (idradm) [reply front/main/app/templates/components/image-review-summary.hbs]: There is nothing about that in the ticket, and as this is a breaking change, we should do it after mercury changes are released.

File: front/main/app/templates/components/image-review-summary.hbs [hunk 2]
```diff
@@ -26,13 +26,15 @@
 		{{#if model.summary}}
 			<table class="image-review-modal-history">
 				<tr>
+					<td>Total Uploaded</td>
 					<td>Total Reviewed</td>
 					<td>Average Per User</td>
```
Comment (sqreek) [inline front/main/app/templates/components/image-review-summary.hbs]: Can we delete it here?

Comment (idradm) [reply front/main/app/templates/components/image-review-summary.hbs]: Ticket mentioned only `distance to average`.

**Pull request merged**